### DISCUSSION TITLE
Says I have no wallet connected but I doooooo!

### DISCUSSION THREAD

**Discussion by burstburst** [btcpayserver/btcpayserver#2487] [Technical Support] [answered]
Hey y'all! Thanks in advance for the help here. We're using BTCpay for checkout on our site and only just noticed an error when the invoice goes to get made:

`{"error":"Warning: No wallet has been linked to your BTCPay Store. See the following link for more information on how to connect your store and wallet. (https://docs.btcpayserver.org/btcpay-basics/gettingstarted#connecting-btcpay-store-to-your-wallet)\n4/24/21 7:43:58 PM: Creation of invoice starting\n4/24/21 7:43:59 PM: BTC_USD: The rating rule is coinaverage(BTC_USD)\n4/24/21 7:43:59 PM: BTC_USD: The evaluated rating rule is ERR_RATE_UNAVAILABLE(coinaverage, BTC_USD)\n4/24/21 7:43:59 PM: BTC_USD: Rate rule error (RateUnavailable)\n4/24/21 7:43:59 PM: BTC_USD: Exception reaching exchange coinaverage (Unauthorized access to the API, premium plan needed)\n"}`

I double checked and the settings are all still the same as they were the last time a tx went through last year (RIP) and I can still see the same wallet is attached. Did i miss a change to some params? I has been a long time since I logged into the backend. I updated the server from the maintenance page but no joy so I'm wondering if I have to redeploy somehow. Any suggestions where I should check next? Thanks!

**Comment by Zaxounette** [comment, marked as answer]
Hey @burstburst !

The error also says that the Coinaverage `Rate rule` failed.
Try changing the exchange you pull the rates from in `Your store name` > `Settings` > `Rates`

**Reply by burstburst** [reply]
thank you!! that must be the param im missing. i dont see a way to change where i pull rates from i wonder if im just running an old version?
![image](https://user-images.githubusercontent.com/640355/115971494-c2bd9500-a516-11eb-9c7c-d7945a2aa072.png)

**Reply by Zaxounette** [reply]
Those are the server settings, what you're looking for is in the specific store settings:

![tgWuqpxKt1](https://user-images.githubusercontent.com/51208677/115971560-66279d80-a549-11eb-980b-eb12c814e99f.png)

**Reply by Zaxounette** [reply]
Also you might want to update again, the rates in the server settings have been moved a while ago AFAIR.

**Reply by burstburst** [reply]
thanks again @Zaxounette !! you were right!! thank you so much, it didn't work until i chose kraken but i'll take it! it does seem to be up to date because it doesnt do anything after the update toast. 
![image](https://user-images.githubusercontent.com/640355/115974106-6bc0bb80-a528-11eb-99e4-1c65ef4bdec2.png)

**Reply by Zaxounette** [reply]
Glad your problem was resolved ! 
For info, the latest version at this time is 1.0.7.2.
You can check at the very bottom of any page to see the version you're running.

**Reply by burstburst** [reply]
oh wow i am hella behind thanks for that really appreciate it. hm, ok, well if its stuck there, i'll probably have to redeploy manually right?

**Reply by Zaxounette** [reply]
Two methods to update here: https://docs.btcpayserver.org/FAQ/FAQ-ServerSettings/#how-to-update-btcpay-server

Keep in mind, if you're way behind, first backup your LN channels if you have some, and check that you have your seed words and all that fun stuff. 
Just in case, you never know.

**Reply by burstburst** [reply]
Amazing, thank you so much!!!

**Reply by Zaxounette** [reply]
Welcome ! 

If you could select this thread as having answered your question, it would be appreciated.
It will let others know your problem has been resolved.
